importing-data-5.html

```diff
@@ -226,30 +318,78 @@
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<img alt="../images/importingDataProjectScreen.jpg" class="align-center" src="images/importingDataProjectScreen.jpg" style="width: 100%;" />
+<img alt="../images/importingData5ProjectScreen.jpg" class="align-center" src="images/importingData5ProjectScreen.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3><a class="anchor" id="import-for"></a>Importing for Another User</h3>
+
+<p>If you are an Administrator user, use the <span class="bold">Import For</span> drop down box to select another user to import the data as. The chosen user will then own this data.</p>
+
+<p>The projects and datasets shown in the drop-down boxes as available to import into, is determined by the permission status of the group being used.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/importingData5ImportFor.jpg" class="align-center" src="images/importingData5ImportFor.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
 
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
+<h3><a class="anchor" id="pyramid"></a>Generation of Pyramid Files After Import</h3>
+
+<p>OMERO will generate an image pyramid for any image that is over 3000 x 3000 pixels, very large confocal scans, images captured by very large detectors and for certain file formats such as .lsm, .jpeg or .tiff.</p>
```

Comment:
I don't know the exact size, but it *never* generates pyramids for not-big images, and *only* generates pyramids for big images *if* Bio-Formats cannot read the subresolutions directly out of the image file. This is part of what the OMERO 5 "no data duplication" was about. So the stuff after the first comma is probably examples of the thing before it rather than extra things in addition.